tests_metricflow/query_rendering/test_conversion_metric_rendering.py

```diff
@@ -164,3 +164,31 @@ def test_conversion_metric_with_window_and_time_constraint(
         dataflow_plan_builder=dataflow_plan_builder,
         query_spec=parsed_query.query_spec,
     )
+
+
+@pytest.mark.sql_engine_snapshot
+def test_conversion_metric_with_filter(
```

Comment:
Can we add a similar test for a windowed conversion metric? I think that requires a group by (right?) so would want a different group by than the one we're filtering on